common.blocks/i-bem/__dom/i-bem__dom.js

```diff
@@ -723,13 +723,21 @@ var DOM = BEM.decl('i-bem__dom',/** @lends BEMDOM.prototype */{
             modName = undef;
         }
 
+        names = names.split(' ');
+
         var _self = this.__self,
             selector = '.' +
-                names.split(' ').map(function(name) {
+                names.map(function(name) {
                     return _self.buildClass(name, modName, modVal);
                 }).join(',.'),
             res = findDomElem(ctx, selector);
 
+        (ctx === this.domElem) && names.forEach(function(name) {
+                var key = _self.buildClass(name, modName, modVal),
+                    elem = this._elemCache[key] = res.filter('.' + key);
```

Comment:
definitely — `name + buildModPostfix(modName, modVal)` more like violation of encapsulation of building full class